rust/models/src/host_info.rs

```diff
@@ -43,34 +45,41 @@ pub struct HostInfo {
     alive: u64,
     queued: u64,
     finished: u64,
+    // Hosts that are currently being scanned. The second entry is the host
+    // scan progress. Required for Openvas Scanner type
+    #[cfg_attr(
+        feature = "serde_support",
+        serde(skip_serializing_if = "Option::is_none")
+    )]
+    scanning: Option<HashMap<String, i32>>,
     // Hosts that are currently being scanned. The second entry is the number of
     // remaining VTs for this host.
-    scanning: HashMap<String, usize>,
+    remaining_vts: HashMap<String, usize>,
```

Comment:
The comment indicates that the values holds currently running scans, while the name indicates that those are vts that don't have run yet. Which one is it?